pkg/flowkit/util/utilities.go

```diff
@@ -154,3 +156,45 @@ func TestnetFaucetURL(publicKey string, sigAlgo crypto.SignatureAlgorithm) strin
 func MainnetFlowPortURL(publicKey string) string {
 	return fmt.Sprintf("%s%s", FlowPortUrl, strings.TrimPrefix(publicKey, "0x"))
 }
+
+func DirExists(path string) bool {
+	info, err := os.Stat(path)
+	if os.IsNotExist(err) {
+		return false
+	}
+	return !info.IsDir()
+}
+
+type ReaderWriter interface {
+	ReadFile(source string) ([]byte, error)
+	WriteFile(filename string, data []byte, perm os.FileMode) error
+}
+
+// AddToGitIgnore adds a new line to the .gitignore if one doesn't exist it creates it.
+func AddToGitIgnore(
```

Comment:
At first, I was thinking no, and even commented that but then I was thinking more in terms of gain/loss, and I think that loss of not doing that is huge, imagine some cases where people will first init the flow and create accounts, start playing around and only then create a project on git (I know sometimes when I prototype I create git much later), the loss there would be their accounts exposed without maybe even realizing it. The gain of not doing it would be not creating an additional file, and although I agree that creating some files in the background is not great, we are actually not doing that since we are telling the user we will do that. I feel if we prevent one case of posting this account to git is worth it. I was actually thinking we will keep the encryption tbh and add the to a separate file.

Reply:
That reasoning makes sense to me! 👍